verification/curator-service/api/src/util/validate-env.ts

```diff
@@ -61,36 +75,28 @@ export default function validateEnv(): Readonly<{
             desc: 'OAuth client secret from the Google developer console',
             devDefault: 'replace to enable auth',
         }),
+        MAPBOX_PERMANENT_GEOCODE: bool({
+            desc: 'Whether to use the permanent geocode endpoint',
+            devDefault: false,
+            default: true,
+        }),
+        MAPBOX_TOKEN: str({
+            desc: 'Mapbox token to use for geocoding',
+            devDefault: '',
+        }),
+        PORT: port({ default: 3001 }),
+        SERVICE_ENV: str({
+            choices: ['local', 'dev', 'prod'],
+            desc: 'Environment in which the service is running',
```

Comment:
don't you need to edit the environment vars in kubernetes as well? plan to do it in a later PR?

Reply:
Are the changes in `curator.yaml` sufficient?

Reply:
ohh, yeah sorry I missed that, nice use of metadata.labels.environment here